resources/crds/ripsaw_v1alpha1_kube-burner_cr.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: kube-burner-cluster-density
+  namespace: my-ripsaw
+spec:
+  # Metadata information
+  elasticsearch:
+    server: <ES_URL>
+    port: <ES_PORT>
+  prometheus:
+    # Elastic search instance with full URL format. https://elastic.apps.org:9200
+    server: <ES_URL>
+    # Prometheus bearer token
+    prom_token: <PROMETHEUS_BEARER_TOKEN>
+    # Prometheus URL with full URL format. https://prometheus-k8s.openshift-monitoring.svc.cluster.local:9091
+    prom_url: <PROMETHEUS_URL>
+  workload:
+    name: kube-burner
+    args:
```

Comment:
We might want to add an option to delete/re-use the namespace/objects under test if they already exist. Thoughts?